src/core_init_atmosphere/mpas_init_atm_cases.F

```diff
@@ -2185,13 +2163,15 @@ subroutine init_atm_case_mtn_wave(dminfo, mesh, nCells, nVertLevels, state, diag
             do k=1,nz1
                ztemp = .25*( zgrid(k,cell1 )+zgrid(k+1,cell1 )  &
                             +zgrid(k,cell2)+zgrid(k+1,cell2))
-               u(k,i) = um
-               if(i == 1 ) u_init(k) = u(k,i) - us
-#ifdef ROTATED_GRID
-               u(k,i) = sin(angleEdge(i)) * (u(k,i) - us)
-#else
-               u(k,i) = cos(angleEdge(i)) * (u(k,i) - us)
-#endif
+               u(k,i) = vm
+
+               if(i == 1 ) then 
+                 v_init(k) = vm
+                 u_init(k) = 0.
+               end if
```

Comment:
It might be cleaner to move this block of code out of the loop over `nEdges`, since `u_init` and `v_init` are independent of horizontal location. We could also set `u_init(:)` to `um` rather than `0.`.

Reply:
This code has been moved outside the nEdges loop, as suggested.